### PR TITLE
[V4] Feat: Create shared wallets

### PR DESCRIPTION
**Pull request by Gamboster**

### CODE REVIEW

**Comment by JDonadio** [comment]
If you create a shared wallet but just go back (from copayer page to home) then you cannot reach the copayer page anymore.

**Comment by JDonadio** [comment]
Clicking in the invitation code display the toast message twice.

File: src/pages/add/copayers/copayers.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <ion-title *ngIf="wallet">{{ wallet.name }}</ion-title>
+  </ion-navbar>
+
+</ion-header>
+
+
+<ion-content>
+  <div *ngIf="wallet" [hidden]="wallet.notAuthorized">
+    <ion-list class="copayers-secret">
+      <ion-item translate>
```
Comment (JDonadio) [inline src/pages/add/copayers/copayers.html]: Translate directive should be used as pipe, not in ion labels.

File: src/pages/add/copayers/copayers.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    private walletProvider: WalletProvider
+  ) {
+
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad CopayersPage');
+    this.wallet = this.profileProvider.getWallet(this.navParams.data.walletId);
+    this.updateWallet();
+    this.shareIcon = this.platformProvider.isIOS ? 'iOS' : 'Android';
```
Comment (JDonadio) [inline src/pages/add/copayers/copayers.ts]: Is it used?
We should use one, not depending on the platform.

Comment (Gamboster) [reply src/pages/add/copayers/copayers.ts]: It isn't used at the moment. I will remove it

File: src/pages/add/copayers/copayers.html (hunk 1 above)
Comment (Gamboster) [reply src/pages/add/copayers/copayers.html]: Got it. Thanks

**Pull request merged**